images/router/nginx/Dockerfile

```diff
@@ -0,0 +1,31 @@
+#
+# This is the NGINX router for OpenShift Origin.
+#
+# The standard name for this image is openshift/origin-nginx-router
+#
+FROM openshift/origin
+
+RUN INSTALL_PKGS="nginx" && \
```

Comment:
@rajatchopra Just curious here. How do we specify which version of nginx is used?

Reply:
Not specify-able yet. This is the base image being built. And we will pick the standard install from openshift/origin image's configured repositories.